rfcs/0013-address-encoding.md

```diff
@@ -0,0 +1,54 @@
+## Summary
+[summary]: #summary
+
+Change address encoding from Base64 to Base58.
+
+## Motivation
+
+[motivation]: #motivation
+
+First, it's the standard format across other well known cryptocurrencies (Ethereum, Bitcoin). That means most entities in the ecosystem, from end-users to exchanges, will be more familiar and comfortable with Base58 encodings than what we have currently.
```

Comment:
Ethereum uses hexadecimal with a cute parity scheme based on capitalizing some letters.

Reply:
We could invert their capitalization. :-)

Reply:
Good point, @enolan. I agree we should have a distinctive prefix that's human readable, and have incorporated that suggestion into the RFC.